pages/api_integration-trading/isolated_markets.md

```diff
@@ -0,0 +1,66 @@
+# Isolated Markets
```

Comment:
Should we have an overview of what "isolated market" is used for? I think this blog post has a good summary of TLDR, what isolated market is used for, how the insurance fund and collateral pools are separate, etc.

https://dydx.exchange/blog/introducing-isolated-markets-and-isolated-margin